simplified-viewer-epub-readium2/build.gradle

```diff
@@ -10,7 +10,10 @@ dependencies {
   implementation libraries.androidXLifecycle
   implementation libraries.androidXLifecycleViewmodel
   implementation libraries.kotlinStdlib
+  implementation libraries.nyplDRMCore
   implementation libraries.nyplR2API
   implementation libraries.nyplR2Vanilla
   implementation libraries.nyplR2Views
+  implementation libraries.r2Shared
```

Comment:
These two dependencies, `r2Shared` and `r2Streamer`, are provided as transitive dependencies from the `org.librarysimplified.r2.api` module in "Simplified-R2-Android". As a result, I think these are redundant declarations and can be removed.

Shouldn't hurt anything if left here, though.

Reply:
I explicitly added them because some symbols from these modules are directly used in `simplified-viewer-epub-readium2`.
Feel free to remove them if you prefer so.